src/__testing__/Navigation.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import App from '../App';
+
+describe('Test for Nav', () => {
+  test('snapshot for App', () => {
+    const rendererComponent = renderer
+      .create(
+        <Provider store={store}>
+          <App />
+        </Provider>,
+      )
+      .toJSON();
+    expect(rendererComponent).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly implement more tests that test the redux pure functions and your API using mocks, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.) requirement.